plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -1708,6 +1688,47 @@ def email_dest_needs_update(self, email_params, email_dest_detail_in_ccc):
 
         return update_needed
 
+    def check_status_api_events(self, status_execution_id):
```

Comment:
Can't we move this API in base class (dnac.py) ?

Reply:
We can move but this API is only used in Event Management while configuration destination and notification. I don't think it will be used by other modules.

Reply:
The functionality of the API is not specific to the "Event Management while configuration destination and notification" module? This functionality could also be utilized by other modules. Can we write in a shared location?

Reply:
sure, moved the api to dnac.py file.